src/react/actions/sts.js

```diff
@@ -16,8 +16,8 @@ export function assumeRoleWithWebIdentity(
     const { oidc } = getState();
     const assumeRoleParams = {
       idToken: oidc.user.id_token,
-      RoleSessionName: 'app1',
       roleArn,
+      RoleSessionName: oidc.user.profile.sub,
     };
```

Comment:
We should probably try to avoid to duplicate the "logic" here and https://github.com/scality/zenko-ui/pull/203/files#diff-8026add9d0d1ea46017507c189fdfdb7c83af900017db5106efa81c370ed518aR123 versus `getAssumeRoleWithWebIdentityIAM` in IAM client. If you can address it part of this PR that would be awesome, else we can open a debt ticket and address it later.

Reply:
Done: https://github.com/scality/zenko-ui/pull/203/commits/403be56e5e96c5739c486110eda2499bc7d8360b